public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderContainer.test.tsx

```diff
@@ -30,6 +31,96 @@ describe('LokiQueryBuilderContainer', () => {
       refId: 'A',
     });
   });
+  it('uses | to separate multiple values in label filters', async () => {
+    const props = {
+      query: {
+        expr: '{app="app1"}',
+        refId: 'A',
+      },
+      datasource: createLokiDatasource(),
+      onChange: jest.fn(),
+      onRunQuery: () => {},
+      showExplain: false,
+    };
+    props.datasource.getDataSamples = jest.fn().mockResolvedValue([]);
+    props.datasource.languageProvider.fetchSeriesLabels = jest.fn().mockReturnValue({ job: ['grafana', 'loki'] });
+    props.onChange = jest.fn();
+
+    render(<LokiQueryBuilderContainer {...props} />);
+    await userEvent.click(screen.getByLabelText('Add'));
+    const labels = screen.getByText(/Label filters/);
+    const selects = getAllByRole(getSelectParent(labels)!, 'combobox');
```

Comment:
since we click with the select I think we want to use await findBy* here as well?

Reply:
Just to confirm, you mean `const selects = await findAllByRole(getSelectParent(labels)!, 'combobox');`? There shouldn't be anything async, so I am not sure it is need. Can you share why do you think it is needed?

Reply:
see https://github.com/grafana/grafana/pull/79949, I think this can lead to test flake, when interacting with elements that might not be present yet in the DOM we get that `test was not wrapped in act(...).` error

Reply:
👍 I'll update then

Reply:
Fixed in https://github.com/grafana/grafana/pull/79978/commits/a454f452d999e6d5a911344fee436e68cf9b6a7e